canonical-web-and-design/ubuntu.com
# Create /appliance/lxd pages

## Done

- Create /appliance/lxd pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/lxd
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page only provides VM and Raspberry pi 4 instructions (lots of logic and variables for this but apparently it will be reused in the future)
- Refer to the [brief](https://docs.google.com/document/d/1uU1XRM6vYLkB1LwLU9uU1FleJVpydvDGAODL10RdMng/edit?ts=5f50ca45) for copy and yaml


## Issue / Card

Fixes #8356
